pkg/ui/workspaces/cluster-ui/src/insights/workloadInsightDetails/statementInsightDetails.tsx

```diff
@@ -140,9 +145,27 @@ export const StatementInsightDetails: React.FC<
                 <section className={cx("section")}>
                   <Row gutter={24}>
                     <Col span={24}>
-                      <SqlBox
-                        value={explain || "Not available."}
-                        size={SqlBoxSize.custom}
+                      <Loading
+                        loading={
+                          !loadedExplain &&
+                          insightEventDetails?.planGist?.length > 0
+                        }
+                        page={"stmt_insight_details"}
+                        error={explainError}
+                        render={() => (
```

Comment:
I believe I changed the loading component behaviour s.t. we can actually just put this as a direct child of the loading component, instead of having to go through this prop.

Reply:
curious, what would be the benefit of the change?

Reply:
Functionally for this case there would be no difference, was just pointing out that it was possible since it would be equivalent to rendering children (and IMO just looks cleaner as opposed to rendering it in the prop). Up to you if you want to change it or not though!